Layouts/Base.inc

```diff
@@ -315,23 +327,44 @@ $endif
 ;**** **** **** **** ****
 ; ADC and temperature measurement
 ;**** **** **** **** ****
-TEMP_LIMIT		EQU	49			; Temperature measurement ADC value for which main motor power is limited at 80degC (low byte, assuming high byte is 1)
-TEMP_LIMIT_STEP	EQU	9			; Temperature measurement ADC value increment for another 10degC
+; Temperature measurement ADC value for which main motor power is limited at 80degC (low byte, assuming high byte is 1)
```

Comment:
Please, leave TEMP_LIMIT and TEMP_LIMIT_STEP to the old values so temperature related features works for EDT. Explanation below.

Reply:
This is related to the ADC topic, so I'd say we leave that for now as it is...

Reply:
Reopened - see comment above.